.github/workflows/scan_images.yaml

```diff
@@ -0,0 +1,64 @@
+name: Scan Images In Bahmni Namespace
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  scan-images:
+    name: Scan Images In Bahmni Namespace
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get current date
+        id: date
+        run: |
+          date=$(date +'%d-%m-%Y')
+          echo "date=$date" >> $GITHUB_OUTPUT
+      - name: Get Scanner Script and Template File
+        run: |
+          curl -o image-scanner.sh https://raw.githubusercontent.com/Bahmni/bahmni-infra-utils/main/image-scanner.sh
+          chmod +x image-scanner.sh
+          curl -o html.tpl https://raw.githubusercontent.com/Bahmni/bahmni-infra-utils/main/html.tpl
+      - name: Scan Images In Bahmni Namespace
+        run: |
+          ./image-scanner.sh
+      - name: Upload Report Artifact
```

Comment:
This step is not needed. We can directly commit the report to security-private repo from this job